src/services/ceramic-service.ts

```diff
@@ -28,43 +23,11 @@ export default class CeramicService {
    * Sets dependencies
    */
   constructor(@inject('ipfsService') private ipfsService?: IpfsService) {
-    if (config.ceramic.validateRecords) {
-      this._client = new CeramicClient(config.ceramic.apiUrl);
-
-      const keyDidResolver = KeyDidResolver.getResolver();
-      const threeIdResolver = ThreeIdResolver.getResolver(this._client);
-      this._resolver = new Resolver({
-        ...threeIdResolver, ...keyDidResolver,
-      })
-    }
+    this._client = new CeramicClient(config.ceramic.apiUrl);
```

Comment:
The Ceramic service is needed for the `anchor` CAS only and not for the API. If we leave it like this it will create a chicken and egg problem when we need to deploy Ceramic instance and CAS. Ceramic instance needs a CAS api and CAS needs Ceramic API.

Reply:
Not sure I follow this. Can you elaborate @simonovic86 ?

Reply:
Good call, done